### PR TITLE
[Term Entry] Swift .contains() method

### PR DESCRIPTION
**Pull request by Iker1211**
### Description

I write the entry for the .contains() method. I created an .md file in this route: content/swift/concepts/dictionaries/terms/contains/contains.md. My motivation is start to collaborate in GitHub projects. 

### Issue Solved

Write the .contains() entry.
Closes issue https://github.com/Codecademy/docs/issues/5118

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by mamtawardhani** [comment]
Hey @Iker1211 the entry doesn't have any related content. Could you please recheck and update it?

**Comment by Iker1211** [comment]
Is good man

**Comment by mamtawardhani** [comment]
> Is good man

IMG_20240830_223958

This is the only content in the file @Iker1211

**Comment by Iker1211** [comment]
ohh sorry.

**Comment by Iker1211** [comment]
Now is good.

File: content/swift/concepts/dictionaries/terms/contains/contains.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (Iker1211) [inline content/swift/concepts/dictionaries/terms/contains/contains.md:1]: Now its done

**Review by mamtawardhani — APPROVED** [review]
Thank you for your contribution to Codecademy Docs @Iker1211, the entry is nicely written! 

It looks good for a second round of review 🚀

**Pull request merged**